[2024-06-02T15:12:12.269+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: alert.run_detect_anomalies_script scheduled__2024-06-02T01:00:00+00:00 [queued]>
[2024-06-02T15:12:12.309+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: alert.run_detect_anomalies_script scheduled__2024-06-02T01:00:00+00:00 [queued]>
[2024-06-02T15:12:12.314+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-06-02T15:12:12.412+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): run_detect_anomalies_script> on 2024-06-02 01:00:00+00:00
[2024-06-02T15:12:12.443+0000] {standard_task_runner.py:60} INFO - Started process 8433 to run task
[2024-06-02T15:12:12.462+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'alert', 'run_detect_anomalies_script', 'scheduled__2024-06-02T01:00:00+00:00', '--job-id', '734', '--raw', '--subdir', 'DAGS_FOLDER/alert.py', '--cfg-path', '/tmp/tmpbnv4k2_m']
[2024-06-02T15:12:12.472+0000] {standard_task_runner.py:88} INFO - Job 734: Subtask run_detect_anomalies_script
[2024-06-02T15:12:12.806+0000] {task_command.py:423} INFO - Running <TaskInstance: alert.run_detect_anomalies_script scheduled__2024-06-02T01:00:00+00:00 [running]> on host 6f8b2a20b5b0
[2024-06-02T15:12:13.254+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='alert' AIRFLOW_CTX_TASK_ID='run_detect_anomalies_script' AIRFLOW_CTX_EXECUTION_DATE='2024-06-02T01:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-02T01:00:00+00:00'
[2024-06-02T15:12:13.596+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-06-02T15:12:13.673+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=alert, task_id=run_detect_anomalies_script, execution_date=20240602T010000, start_date=20240602T151212, end_date=20240602T151213
[2024-06-02T15:12:13.787+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-06-02T15:12:13.916+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
